core/src/HIP/Kokkos_HIP_Space.cpp

```diff
@@ -590,9 +590,8 @@ namespace Experimental {
 //{ return Impl::HIPInternalDevices::singleton().m_hipDevCount ; }
 
 int HIP::concurrency() {
-  // FIXME_HIP
-  // MI60: ThreadsPerComputeUnit*ComputeUnits/ShaderEngine*ShaderEngines)
-  return 2536 * 16 * 4;
```

Comment:
Now, I see `163840` instead of `162304`  so maybe it is rather 2560 threads/compute unit. Anyway, this is better.